test/net/sourceforge/kolmafia/request/CharPaneRequestTest.java

```diff
@@ -33,6 +34,23 @@ public void beforeEach() {
     CharPaneRequest.reset();
   }
 
+  @Test
+  void canFindAvatarWithCrossorigin() {
```

Comment:
Split this into two tests, each with its own html fixture, rather having both in the same test.

This is especially important because CharpaneRequest sets a LOT of character state, and one hopes that a BeforeEach (and AfterAll) method cleans that up - especially since you don't have Cleanups that do. 😀

Reply:
Done!  

I thought I was checking "what happens if it changes between two charPaneRequests?", but I think this is a better set of unit tests as unit tests.